editor/src/components/editor/store/dispatch.tsx

```diff
@@ -481,29 +481,29 @@ function editorDispatchInner(
     // Tested quickly and it broke selection, but I'm mostly certain
     // it should only merge when both have changed.
     if (metadataChanged) {
-      result = produce(result, (r) => {
-        if (r.editor.canvas.dragState == null) {
-          r.editor.jsxMetadataKILLME = reconstructJSXMetadata(result.editor)
-        } else {
-          r.editor.canvas.dragState.metadata = reconstructJSXMetadata(result.editor)
+      if (result.editor.canvas.dragState == null) {
+        result = {
+          ...result,
+          editor: {
+            ...result.editor,
+            jsxMetadataKILLME: reconstructJSXMetadata(result.editor),
+          },
         }
-        // TODO Re-enable this once we have addressed the root cause of the false positives
-        // (these were firing frequently even on elements that remained > 0x0 dimensions)
-        //
-        // const allLostElements = lostElements(r.editor.selectedViews, r.editor.jsxMetadataKILLME)
-        // const newLostElements = TP.filterPaths(allLostElements, r.editor.warnedInstances)
-        // if (newLostElements.length > 0 && isBrowserEnvironment) {
-        //   // FIXME The above `isBrowserEnvironment` check is required because this is tripped by tests that don't update the metadata
```

Comment:
I can undelete this commented out code, but it felt like dead weight we are bringing along for the ride

Reply:
I'm fine with this being binned off